app/Http/Controllers/Device/Tabs/PortsController.php

```diff
@@ -283,7 +294,7 @@ private function getTabs(Device $device): array
     {
         $tabs = [
             ['name' => __('Basic'), 'url' => 'basic'],
-            ['name' => __('Detail'), 'url' => ''],
+            ['name' => __('Detail'), 'url' => 'detail'],
```

Comment:
This shouldn't be here. Otherwise the URL always has detail in it.

Reply:
I just had a closer look at this, and pushed another change to make the existing option work properly